<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dedicated Counters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Z:/Lab/Andor Project/imaging_software_g/ui/Experiment.qrc">
    <normaloff>:/openicon/icons/gnome-power-statistics.png</normaloff>:/openicon/icons/gnome-power-statistics.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionClear"/>
   <addaction name="actionData_Reporting"/>
   <addaction name="actionPlot_Data"/>
   <addaction name="actionTimeseries"/>
  </widget>
  <widget class="QDockWidget" name="settingsDock">
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Z:/Lab/Andor Project/imaging_software_g/ui/Experiment.qrc">
     <normaloff>:/kde/icons/filesave.png</normaloff>:/kde/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="Z:/Lab/Andor Project/imaging_software_g/ui/Experiment.qrc">
     <normaloff>:/kde/icons/edit-clear.png</normaloff>:/kde/icons/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="Z:/Lab/Andor Project/imaging_software_g/ui/Experiment.qrc">
     <normaloff>:/openicon/icons/media-playback-start-7.png</normaloff>:/openicon/icons/media-playback-start-7.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="Z:/Lab/Andor Project/imaging_software_g/ui/Experiment.qrc">
     <normaloff>:/openicon/icons/media-playback-stop-7.png</normaloff>:/openicon/icons/media-playback-stop-7.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionData_Reporting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/kde/icons/gear.png</normaloff>:/kde/icons/gear.png</iconset>
   </property>
   <property name="text">
    <string>Data Reporting</string>
   </property>
   <property name="toolTip">
    <string>enable reporting of data from the FPGA</string>
   </property>
  </action>
  <action name="actionPlot_Data">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/office-chart-line.png</normaloff>:/openicon/icons/office-chart-line.png</iconset>
   </property>
   <property name="text">
    <string>Plot Data</string>
   </property>
   <property name="toolTip">
    <string>Plot data</string>
   </property>
  </action>
  <action name="actionTimeseries">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/kde/icons/grafanalogo.png</normaloff>:/kde/icons/grafanalogo.png</iconset>
   </property>
   <property name="text">
    <string>Timeseries</string>
   </property>
   <property name="toolTip">
    <string>Log in Timeseries database</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Experiment.qrc"/>
 </resources>
 <connections/>
</ui>
